data/campaigns/Two_Brothers/scenarios/04_Return_to_the_Village.cfg

```diff
@@ -362,7 +362,8 @@ But I am still troubled. I wonder... is this sense of foreboding I feel merely a
 
         [message]
             speaker=Arvith
-            message= _ "I must go back to earning my living. But we have our amulets, little brother. If you are beset again, I will come."
+            # po: "you have my word" is a idiom for promising to do something
```

Comment:
Granted, it's just the `po` hint rather than the actual prose, but I think this should read 'an idiom'.